box.json

```diff
@@ -25,18 +25,29 @@
     "license":[
         {
             "type":"MIT",
-            "URL":""
+            "URL":"https://github.com/jcberquist/stripe-cfml/blob/master/LICENSE"
         }
     ],
-    "contributors":[],
-    "dependencies":{},
+    "contributors":[
+        "jcberquist",
+        "elpete"
+    ],
+    "dependencies":{
+        "str":"^2.0.0"
```

Comment:
So it looks to me like the `str` dependency is only being used to camel case the incoming stripe event names, is that right? I am torn about adding this as a dependency, because I would still like people to be able to "clone and go", and this would require a CommandBox install. On the other hand, presumably if you are using ColdBox, you are using CommandBox as well, and the `Webhooks.cfc` would really only be used in that case, so one could still clone and use the module outside of ColdBox without needing the `str` dependency, correct?

That said, since a simple regex replace would suffice to convert the events, could we just do that instead?

Reply:
People should be able to clone and go without trouble still.  If your regex works, though, then that's fine.  I'll test it and add it.